frontend/src/containers/CamListContainer/CamListContainer.js

```diff
@@ -2,6 +2,8 @@ import React, { Component } from "react";
 import classes from "./CamListContainer.module.css";
 import Discover from "../../components/Discover/Discover";
 import CamListItem from "../../components/CamListItem/CamListItem";
+import CameraDetails from "../../components/CameraDetails/CameraDetails";
+import Grid from "@material-ui/core/Grid";
 
 const BASE_URL = "http://localhost:8080/api/v1/cameras";
```

Comment:
http://localhost:8080 ne legyen beégetve
.env
https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
javítottam, becsekoltam a .env fájlt, így mostmár ott megadható, hogy hol fut a backend.